ee/clickhouse/queries/test/test_experiments.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from ee.clickhouse.queries.experiments.funnel_experiment_result import validate_event_variants
+from rest_framework.exceptions import ValidationError
+
+
+class TestExperiments(unittest.TestCase):
+    def test_validate_event_variants(self):
+        expected_message = "No experiment events have been ingested yet"
+        with self.assertRaises(ValidationError) as context:
+            validate_event_variants([], ["test", "control"])
+
+        self.assertIn(expected_message, str(context.exception))
+
+    def test_validate_event_variants_2(self):
+        filtered_results = [
```

Comment:
Can we test this using real possible results from a funnel query? 🤔  Not sure such a result is ever possible

Reply:
Correct, this isn't possible. If step 1 is not present at all, the funnel results will be empty 👍